faqs/galaxy/analysis_regular_expressions.md

```diff
@@ -43,11 +43,13 @@ Below are just a few examples of basic expressions:
 | `chr\d{1,2}`         | `chr` followed by 1 or 2 digits |
 | `.*abc$`             | anything with `abc` at the end of the line |
 | `^$`                 | empty line |
+| `^>.*`			   | Line starting with `>` (e.g. Fasta header) |
```

Comment:
@Delphine-L what do you think about adding `regular expression` to this list in Galaxy https://github.com/galaxyproject/galaxy/pull/18722/files#diff-1832837c9e0fd387cc997af7d63fa5e92dc1e50a311e36dd09f8536469747d6c

Explain what that term is and add a link to this FAQ?

ping @jmchilton as well for input